### PR TITLE
404 error page

### PR DESCRIPTION
**Pull request by ShreyaDhiman24**
## Related Issue
Closes: #452 
## Description of Changes

image

## Checklist:
- [x]  My code adheres to the established style guidelines of this project.
- [x]  I have conducted a self-review of my code.
- [x]  I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

### CODE REVIEW

**Comment by anmode** [comment]
Screenshot 2023-07-01 at 7 59 10 PM
do not include header! and nav

**Comment by anmode** [comment]
cool design!

**Comment by ShreyaDhiman24** [comment]
> Screenshot 2023-07-01 at 7 59 10 PM do not include header! and nav

Check this out

**Comment by anmode** [comment]
1. change the dummy text 
2. Optimize mobile mode please!

**Comment by anmode** [comment]
remove error.png as you have error.svg

**Comment by anmode** [comment]
last fix CI- `npx prettier --write .`
In your local

**Comment by ShreyaDhiman24** [comment]
> npx prettier --write

yeah done

File: pages/404.page.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import styles from "../styles/404.module.css";
+import dynamic from "next/dynamic";
+const Footer = dynamic(() => import("../components/Footer"));
```
Comment (anmode) [inline pages/404.page.js:6]: invalid path!

Comment (ShreyaDhiman24) [reply pages/404.page.js:6]: > invalid footer path!

It's valid. or don't want footer in error page ?

**Comment by anmode** [comment]
button effect is not good, see your previous change which you showed in description

**Comment by ShreyaDhiman24** [comment]
> button effect is not good, see your previous change which you showed in description

done or not?

**Pull request merged**